ibis/expr/signature.py

```diff
@@ -81,6 +82,11 @@ def Argument(validator, default=EMPTY):
         If callable is passed, it will be executed just before the inner,
         and itsreturn value will be treaded as default.
     """
+    warnings.warn(
+        "ibis.expr.signature.Argument will be removed in ibis 3.0",
+        FutureWarning,
```

Comment:
I don't think this is necessary if we bump straight to 3.0

Reply:
Will we bump straight to 3.0?

Reply:
I think we should

Reply:
Ok, then I'm removing the deprecation warning.